components/InventoriedResources.tsx

```diff
@@ -0,0 +1,16 @@
+import Link from "next/link";
+import Card from "./Card";
+import React from "react";
+
+const InventoriedResources = ({inventoriedResources}:{inventoriedResources:any}) => {
```

Comment:
Why `any` is needed? Are there different types supported? Coulden't have [Unions](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types) in that case?